cupy/testing/helper.py

```diff
@@ -109,6 +109,24 @@ def test_func(self, *args, **kw):
 
 def numpy_cupy_allclose(rtol=1e-7, atol=0, err_msg='', verbose=True,
                         name='xp', type_check=True, accept_error=True):
+    """Decorator that checks NumPy results and CuPy ones are close.
+
+    Args:
+         rtol(float): Relative torelance.
+         atol(float): Absolute torelance
+         err_msg(str): The error message to be printed in case of failure.
+         verbose(bool): If True, the conflicting values are
+             appended to the error message.
+         name(str): Argument name whose value is either ``numpy`` or ``cupy``.
```

Comment:
I do not think we need quotations because the type is not `str` (`'numpy'`) but module (`numpy` module).